src/coreclr/build-runtime.cmd

```diff
@@ -77,7 +77,8 @@ set __CrossArch2=
 set __CrossOS=0
 set __PgoOptDataPath=
 set __CMakeArgs=
-set __Ninja=0
+set __Ninja=1
+set __NinjaOptInDeprecated=0
```

Comment:
curious, what is this used for?

Reply:
I'll go remove that. I was initially going to have a deprecation message printed when the -ninja flag was used but I decided to just let it be a no-op at the individual script level.